app/controllers/api_controller/parser.rb

```diff
@@ -197,6 +201,7 @@ def validate_method_action(method_name, action_name)
       cspec = collection_config[cname.to_sym]
       target = request_type_target.last
       aspec = cspec["#{target}_actions".to_sym]
+      return unless aspec
```

Comment:
for all methods ?   let's do this *only* for :get method_name, i.e. if no action specs defined for get, let's allow, others are required.